py-morse-code/morse_code.py

```diff
@@ -0,0 +1,69 @@
+from node import Node
+
+
+class MorseCode:
+
+    _code_map = {
+        " ": "  ",
```

Comment:
all these values have a blank space at the end. i see how it's used below, but i probably wouldn't tie the implementation details to the raw values.

Then you can optimize this code:
```python
# old
if v == (c + " "):

# new
if v == c:
```